test/mocks/upstream/cluster_info.h

```diff
@@ -130,6 +131,8 @@ class MockClusterInfo : public ClusterInfo {
   Upstream::TransportSocketMatcherPtr transport_socket_matcher_;
   NiceMock<Stats::MockIsolatedStatsStore> load_report_stats_store_;
   ClusterLoadReportStats load_report_stats_;
+  NiceMock<Stats::MockIsolatedStatsStore> timeout_budget_stats_store_;
```

Comment:
Delete this.

Reply:
Not going to delete this -- turns out it's way harder to assert on the store for the histograms when they share an underlying store!